addOns/grpc/src/main/java/org/zaproxy/addon/grpc/internal/HttpPanelGrpcView.java

```diff
@@ -145,8 +145,6 @@ public void dataChanged(HttpPanelViewModelEvent e) {
             }
         } catch (Exception er) {
             httpPanelGrpcArea.setText(er.getMessage());
-            httpPanelGrpcArea.setBorder(BorderFactory.createLineBorder(Color.RED));
-            httpPanelGrpcArea.setBorder(null);
```

Comment:
You could keep the border but you would have to reset it when the `dataChanged` method is called rather than immediately after.